edb/lib/std/50-constraints.edgeql

```diff
@@ -46,6 +46,7 @@ std::expression EXTENDING std::constraint
 CREATE ABSTRACT CONSTRAINT
 std::exclusive EXTENDING std::constraint
 {
+    SET is_aggregate := true;
```

Comment:
We don't formally use "aggregate" in EdgeQL, we have `SET OF` for that. Maybe `is_setof` would work better here?

Reply:
@elprans

Reply:
Well, `SET OF` is the cardinality modifier on the argument/return type, but we _do_ call these "aggregate parameters" and "aggregate functions": https://edgedb.com/docs/edgeql/overview#aggregates.

Reply:
Alright.